cfme/ansible/repositories.py

```diff
@@ -75,6 +75,16 @@ def is_displayed(self):
         )
 
 
+class PlaybookRepositoryView(RepositoryDetailsView):
```

Comment:
Optional:
I think the name should be `RepositoryPlaybooksView`.... as its playbooks of specific repositories